rest/taskrouter/statistics/worker/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,25 @@
+// Download the twilio-csharp library from
+// https://www.twilio.com/docs/libraries/csharp#installation
+using Newtonsoft.Json.Linq;
+using System;
+using Twilio;
+using Twilio.Rest.Taskrouter.V1.Workspace.Worker;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        const string workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string WorkerSid = "WKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+        var stats = WorkerStatisticsResource.Fetch(workspaceSid, WorkerSid);
+        var cumulativeStats = JObject.FromObject(stats.Cumulative);
+
+        Console.WriteLine(cumulativeStats["reservations_accepted"].Value<int>());
```

Comment:
There is no additional value in casting `reservations_accepted` to an `int`. When you pass anything to a `Console.WriteLine` it will be converted to a string. So that this one should be:

```csharp
Console.WriteLine(cumulativeStats["reservations_accepted"]);
```